packages/astro/src/runtime/server/render/util.ts

```diff
@@ -104,6 +104,11 @@ Make sure to use the static attribute syntax (\`${key}={value}\`) instead of the
 		return markHTMLString(` class="${toAttributeString(value, shouldEscape)}"`);
 	}
 
+	// prevent URLs in `content` attributes being escaped for multiple params to work
+	if (key === 'content' && typeof value === 'string' && URL.canParse(value)) {
```

Comment:
Not familiar with this part of the codebase, but can  you for hrefs attributes as well?